README.md

```diff
@@ -1 +1,56 @@
 # Canopy Frontend RFC
+The Canopy Frontend RFC (Request for Comments) is a git repository where the technologies and processes that apply to all
+frontend squads at Canopy are described.
+
+## Purpose
+1. Provide a way to enact change in Canopy's frontend that impacts all projects and squads.
+2. Clarify what things we are aligned on across the entire front-end, so that everything else can be decided by the members of each squad.
+
+## What this isn't
+1. An infringement on the autonomy of squads. The idea is that if we're aligned on the few important things, we can be autonomous on everything else.
+2. Something oppressively prescriptive.
+3. A technical review process where people get feedback on difficult problems they're facing. Although important, the RFC is not how
+   technical reviews happen.
+
+## Signs that this is working
+1. The content is up-to-date and people care about it.
+2. There are regular updates.
```

Comment:
I'm curious to see how often this changes in practice.

I think that there's the potential for there not to be any changes for a decent amount of time, but that it wouldn't necessarily indicate that the process isn't working. Maybe it actually indicates that the process worked so well that it is currently meeting everyone's needs/desires?

Reply:
Probably true. Maybe item `1.` is really what I was wanting to express. I'll change this